CorsixTH/Lua/dialogs/resizables/map_editor.lua

```diff
@@ -1109,9 +1109,6 @@ function UIMapEditor:onMouseDown(button, xpos, ypos)
       repaint = true
     end
 
-  elseif self.cursor.state == "right" then
```

Comment:
Verify the position of this `elseif` doesn't matter to the subsequent `elseif`

Reply:
I can't confirm it in game (can't trigger this clause if I reinstate it with a print), but yes, as it won't check the next elseif.

Reply:
I've reread it again. Was probably late when I did.
The removal of this check is fine